lib/sycamore/sycamore/tests/integration/transforms/test_summarize_images.py

```diff
@@ -1,7 +1,8 @@
 import sycamore
 from sycamore.transforms.partition import ArynPartitioner
-from sycamore.transforms.summarize_images import SummarizeImages
+from sycamore.transforms.summarize_images import LLMImageSummarizer, SummarizeImages
 from sycamore.tests.config import TEST_DIR
+from sycamore.llms.bedrock import BedrockModels, Bedrock
 
 
 def test_summarize_images():
```

Comment:
Mind making this explicitly `test_summarize_images_openai` and suggest explicitly passing `llm=OpenAI...`